estate/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Real Estate',
+    'depends': ['base', 'website'],
+    'category': 'Real Estate/Brokerage',
+    'summary': 'Real estate internal machinery',
```

Comment:
```suggestion
    'summary': "Real estate internal machinery",
```
Similarly change everywhere.